src/components/App.jsx

```diff
@@ -1,7 +1,36 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import QuestionsList from './QuestionsList';
+import AddQuestionBtn from './AddQuestionBtn';
+import getProductQuestions from '../api/getProductQuestions';
+import getProductInfo from '../api/getProductInfo';
 
 function App() {
+  const [productId, setProductId] = useState(3);
+  const [questions, setQuestions] = useState([]);
+  const [productName, setProductName] = useState('');
```

Comment:
I don't know what the "best practice" is for functions like this.  I've only ever written them into the file, but maybe it's better to import them like this?  we should ask which (if any) is best next week